Sources/UIPanGestureRecognizer.swift

```diff
@@ -11,8 +11,8 @@ import Foundation
 open class UIPanGestureRecognizer: UIGestureRecognizer {
     private var initialTouchPoint: CGPoint?
 
-    private var previousTouchesMovedTimestamp: TimeInterval?
-    private var touchesMovedTimestamp: TimeInterval?
+    internal var previousTouchesMovedTimestamp: TimeInterval?
+    internal var touchesMovedTimestamp: TimeInterval?
 
```

Comment:
Is there a way of avoiding changing those to `internal`? The reason I wanted to access them was to simulate a velocity scroll in `testIsDecelerating`

Reply:
see comment in the test :)